extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -329,12 +330,16 @@ export class AmpStoryBookend extends AMP.BaseElement {
   }
 
   /**
-   * Reacts to desktop state updates.
-   * @param {boolean} isDesktop
+   * Reacts to UI state updates.
+   * @param {!UIType} uiState
    * @private
    */
-  onDesktopStateUpdate_(isDesktop) {
-    this.toggleDesktopAttribute_(isDesktop);
+  onUIStateUpdate_(uiState) {
+    this.mutateElement(() => {
```

Comment:
Do you want to set the `[scroll]` attribute here (and in the other shadow roots) so that it doesn't require any JS changes to style those components within the new experiment?

Reply:
Whenever we need to, yes. Just like we do for `desktop` right now actually.
Hopefully the only component that will look different will be the system-layer though, we should be able to keep that simple and relatively centralized to the `amp-story-scroll.css` file.